pkg/scheduler/framework/plugins/imagelocality/image_locality.go

```diff
@@ -79,7 +79,8 @@ func New(_ runtime.Object, h framework.FrameworkHandle) (framework.Plugin, error
 
 // calculatePriority returns the priority of a node. Given the sumScores of requested images on the node, the node's
 // priority is obtained by scaling the maximum priority value with a ratio proportional to the sumScores.
-func calculatePriority(sumScores int64) int64 {
+func calculatePriority(sumScores int64, noContainers int) int64 {
```

Comment:
lets use numContainers, noContainers is really not adequate

Reply:
done